src/main/resources/db/changelog/data/users.csv

```diff
@@ -0,0 +1,4 @@
+id,user_name,email,password,user_status
+1,user one,user@gmail.com,Password1,verified
+2,user two,usertwo@gmail.com,Second45password,verified
+3,userthree,userthree@gmail.com,ThirdPassword3,verified
```

Comment:
should be "user three" here.
And by the way, maybe instead of using "User One" etc. let's use:
- "Alpha User"
- "Beta User"
- "Gamma User"

Later, it may be easier to communicate with each other by voice and describe specifically which user is in question.